src/components/MonitorList.vue

```diff
@@ -243,7 +286,14 @@ export default {
          * @returns {void}
          */
         updateFilter(newFilter) {
-            this.filterState = newFilter;
+            this.$router.replace({
+                query: {
+                    ...this.$route.query,
+                    status: newFilter.status,
+                    active: newFilter.active,
+                    tags: newFilter.tags,
```

Comment:
I think it would feel more intentional if we were to only add the filters when they are nessesary. What do you think?

![image](https://github.com/user-attachments/assets/e54fac0f-8959-4a32-bead-25a3ea9d7c5f)

Reply:
The simplest solution seems to be to set the default value to empty array instead of null like
```
let status = this.$route.query["status"] || [];
```
But this relies on vue (or vue-router) bahaviour. Do you think it would be better to manually `delete` filters with falsy values? Something like:
```
let newQuery = {
    ...this.$route.query,
};
for (const [ key, value ] of Object.entries(newFilter)) {
    if (value === null || value.length === 0) {
        delete newQuery[key];
    } else {
        newQuery[key] = value;
    }
}
 ```

Reply:
> But this relies on vue-router bahaviour

"relying" on this behaviour for this part is fine.

Reply:
Reverted to the first approach